rclcpp/include/rclcpp/create_subscription.hpp

```diff
@@ -107,15 +84,18 @@ create_subscription(
               " ms");
     }
 
-    std::shared_ptr<Publisher<statistics_msgs::msg::MetricsMessage>> publisher =
-      create_publisher<statistics_msgs::msg::MetricsMessage>(
-      node,
+    // TODO(ivanpauno): This could have topics statistics enabled, but I'm not sure if it makes
```

Comment:
I'm confused. Do you mean the topic statistics publisher could have QoS overrides enabled?

I'm not sure. I think it should be discussed in the design doc ultimately and we can revisit it if/when it is documented there.

Reply:
Deleted the comment in https://github.com/ros2/rclcpp/pull/1408/commits/f1062cb78d12b04f5893776284a07bd01d0e9c1c.

> I'm confused.

I'm reading the comment and I'm confused too :joy:.
The comment was about being able to reconfigure topic statistics qos, but we can discuss where to enable qos reconfigurability in a follow-up PR.